doc/sphinx_source/mainDocs/tcl-commands.rst

```diff
@@ -3311,6 +3311,16 @@ The following is a list of bind types and how they work. Below each bind type is
 
   Module: core
 
+(50) AWY3 (stackable)
+
+  bind awy3 <flags> <mask> <proc>
+
+  procname <from> <msg>
```

Comment:
arguments should be identical to other binds:
`procname <nick> <host> <hand> <chan> <awaymsg>` (awaymsg is "" if they return)
and call once per channel the user is on

Reply:
```
.tcl proc foo {nick user hand chan msg} {putlog "nick: $nick user: $user hand: $hand chan: $chan msg: $msg"}
.tcl bind * * foo

[05:10:08] nick: foo, user: foo!foo@foo.com hand: Foo chan: #eggdroptest msg: :This is a test message
[05:10:08] foo!foo@foo.com is now away: This is a test message
[05:10:12] nick: foo, user: foo!foo@foo.com hand: Foo chan: #eggdroptest msg: 
```
Adding a channel mask, bot is on two channels #eggdroptest and #eggtest:
```
.tcl bind awy3 * "#eggdroptest *" goo
.tcl proc goo {nick user hand chan text} {putlog "This was activated by $chan"}

[05:14:49] This was activated by #eggdroptest
[05:14:49] nick: foo, user: foo!foo@foo.com hand: Foo chan: #eggdroptest msg: :Going away now
[05:14:49] nick: foo, user: foo!foo@foo.com hand: Foo chan: #eggtest msg: :Going away now
```